### PR TITLE
Create get recipes in specific kitchen

### PR DESCRIPTION
**Pull request by devin-mcewan**
CARD: https://github.com/orgs/HTMElephant/projects/2/views/1?pane=issue&itemId=49190278

Created a new GET endpoint in /v1/kitchen
  - "/:id/recipes"
  - created SQL query to return all the recipes associated with the Kitchen's Id from req.params. 
  - Returned the recipes.

### CODE REVIEW

File: db/get_all_recipes_by_kitchen_id.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+SELECT r.*
+FROM kitchen_recipes kr 
+JOIN recipes r on r.id = kr.recipe_id
+WHERE kr.kitchen_id = ${id};
```
Comment (MattGVO) [inline db/get_all_recipes_by_kitchen_id.sql:4]: A more common way to write this is to have the `FROM` be the table that you selecting the bulk of the columns from 
```suggestion
SELECT r.*
FROM recipes r 
JOIN kitchen_recipes kr on kr.recipe_id = r.id
WHERE kr.kitchen_id = ${id};
```

Comment (devin-mcewan) [reply db/get_all_recipes_by_kitchen_id.sql]: Resolved!

**Review by jacobdman — APPROVED** [review]
Great work buddy

File: routes/v1/kitchen/kitchen.js [hunk 2]
```diff
@@ -2,6 +2,39 @@ const express = require("express");
...
+  } catch (err) {
+    next(err);
+  }
+});
+router.get("/:id/users", async (req, res, next) => {
+  try {
+    const db = req.app.get("db");
+    const { id } = req.params;
+    const response = await db.get_all_users_by_kitchen_id({ id });
+    console.log('response from db', response)
```
Comment (KyleD8n) [inline routes/v1/kitchen/kitchen.js:20]: There is still a console.log in the code.

File: db/get_all_recipes_by_kitchen_id.sql
Comment (devin-mcewan) [inline db/get_all_recipes_by_kitchen_id.sql:1]: needed this for another card.

File: db/get_all_users_by_kitchen_id.sql
Comment (devin-mcewan) [inline db/get_all_users_by_kitchen_id.sql:1]: Updated SQL to be accurate with Jacob.

File: db/get_kitchen_by_id.sql
Comment (devin-mcewan) [inline db/get_kitchen_by_id.sql:1]: needed this for another card. Was blocking the card I was working on.

File: routes/v1/kitchen/kitchen.js
Comment (devin-mcewan) [inline routes/v1/kitchen/kitchen.js:1]: Added routes for the above SQL.

**Review by devin-mcewan — COMMENTED** [review]
Looks good (yeah it does)

**Review by KyleD8n — APPROVED** [review]

**Pull request merged**